File: backend/src/main/java/com/ll/nbe342team8/domain/order/detailOrder/dto/DetailOrderDto.java
package com.ll.nbe342team8.domain.order.detailOrder.dto;

import com.ll.nbe342team8.domain.order.detailOrder.entity.DeliveryStatus;

public record DetailOrderDto(
        Long orderId,
        Long bookId,
        int bookQuantity,
        DeliveryStatus deliveryStatus
) {
}

File: backend/src/main/java/com/ll/nbe342team8/domain/order/detailOrder/dto/DetailOrderDto.kt
package com.ll.nbe342team8.domain.order.detailOrder.dto

import com.ll.nbe342team8.domain.order.detailOrder.entity.DeliveryStatus

data class DetailOrderDto(
    val orderId: Long,
    val bookId: Long,
    val bookQuantity: Int,
    val deliveryStatus: DeliveryStatus,
    val  coverImage: String
)